geoips/commandline/geoips_command.py

```diff
@@ -52,6 +52,58 @@ def __init__(self):
 plugin_packages = PluginPackages()
 
 
+class ParentParsers:
+    """Object containing shared arguments for commands in a hierarchical order.
+
+    The name of the parsers in this class directly link to the name of the top-level
+    command which will share its arguments. For example, since our top-level command is
+    'geoips' (see GeoipsCLI), then the arguments created by 'geoips_parser' will be
+    shared amongst all of its child commands. As such, every argument created by
+    'list_parser' will be shared by 'list' commands, and so on. So long as the name of
+    the class matches the structure <class_name>_parser, then the arguments will be
+    shared correctly.
+    """
+
+    geoips_parser = argparse.ArgumentParser(add_help=False)
+    geoips_parser.add_argument(
+        "--log_level",
+        "-log",
+        type=str,
+        default="interactive",
+        choices=["debug", "error", "info", "interactive", "warning"],
+        help="The logging level to use for output via the CLI.",
+    )
```

Comment:
I don't see `--log-level` in the output from `geoips -h`.

```
> geoips -h
usage: geoips [-h] {config,get,list,run,test,validate} ...

positional arguments:
  {config,get,list,run,test,validate}
                        geoips instructions.
    config              Various configuration-based commands for setting up your geoips environment. Currently supports `geoips config install <test_dataset_name>` command.
    get                 Retrieve an appropriate GeoIPS artifact. Currently supported `get` calls are: ["family", "interface", "package", "plugin"]
    list                List off available GeoIPS artifacts provided further commands. Current artifacts available for listing are: ["interface", "interfaces", "packages", "plugins", "scripts", "test-datasets", "unit-tests"].
    run                 Run a specified geoips process workflow (procflow) plugin. See 'geoips run single_source' or 'geoips run data_fusion' for more information of what arguments to providel
    test                Run tests implemented in GeoIPS or a separate GeoIPS Package. This could be running a set of linters on a specific package, running a certain integration or normal test script, or running unit tests. Current options are ["linting", "script", "unit-test"].
    validate            Validate a GeoIPS Plugin found at <file_path>. While this is done under the hood via GeoIPS, this is an easy way to test whether or not the plugin you are developing is valid.

options:
  -h, --help            show this help message and exit
```

I do see it in the output from `geoips list -h`, though.

Reply:
Addressing this now. Easy fix.